.gitignore

```diff
@@ -19,3 +19,8 @@ datasets/**
 !datasets/README.md
 .last_update_sha
 temp.tar
+
+.pgpass
```

Comment:
These 4 files are normally in the user's home folder and so should never appear in this repo. Good to be sure they don't get committed by accident, however